### PR TITLE
Add README.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Added a README (HTTP endpoint listings, some examples).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,178 @@
-# bedrock-vc-issuer
-Bedrock Verifiable Credentials issuer software
+# Bedrock Verifiable Credentials Issuer API module _(bedrock-vc-issuer)_
+
+[![Build Status](https://img.shields.io/github/workflow/status/digitalbazaar/bedrock-vc-issuer/Node.js%20CI)](https://github.com/digitalbazaar/bedrock-vc-issuer/actions?query=workflow%3A%22Node.js+CI%22)
```
Comment (davidlehn) [inline README.md:3]: Fix workflow name:
```suggestion
[![Build Status](https://img.shields.io/github/workflow/status/digitalbazaar/bedrock-vc-issuer/Bedrock%20Node.js%20CI)](https://github.com/digitalbazaar/bedrock-vc-issuer/actions?query=workflow%3A%22Bedrock+Node.js+CI%22)
```

Comment (dmitrizagidulin) [reply README.md:3]: Whoops thanks.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,178 @@
...
+```json
+{
+  "credentialId": "...",
+  "credentialStatus": {
+    "type": "RevocationList2020Status"
+  }
+}
+```
+
+`HTTP 200 OK`
```
Comment (davidlehn) [inline README.md:138]: Use same style as below (or could switch to this style below)
````suggestion
```
HTTP 200 OK
```
````

**Review by dlongley — APPROVED** [review]
Thanks! Again, this will need to be updated in the near future, but good to have something to build off of.

**Pull request merged**